src/components/NavigationBar/index.js

```diff
@@ -198,6 +198,9 @@ class NavigationBar extends Component {
       searchSelectWidth: this.getSelectMenuWidth(this.props.searchType),
       showSearchBar: !isMobileView(1000),
       rightContainer: true,
+      showMenu: true,
```

Comment:
The approach is not correct and scalable. The Menu component should be implemented as Stateful and shouldn't require state declaration in the parent.